pylint/extensions/docparams.py

```diff
@@ -83,6 +83,12 @@ class DocstringParameterChecker(BaseChecker):
                   # we can't use the same old_name for two different warnings
                   # {'old_names': [('W9009', 'missing-yields-doc')]},
                  ),
+        'W9015': ('"%s" differing in parameter documentation',
```

Comment:
Since I can't use the same old name (aka `{'old_names': [('W9003', 'missing-param-doc')]}`), how should this be changed?